pkg/apis/packet/types_controlplane.go

```diff
@@ -23,4 +23,11 @@ import (
 // ControlPlaneConfig contains configuration settings for the control plane.
 type ControlPlaneConfig struct {
 	metav1.TypeMeta
+
+	Persistence *Persistence `json:"persistence,omitempty"`
+}
+
+// Persistence contains configuration options for shoot persistent storage.
+type Persistence struct {
+	Enabled *bool `json:"enabled,omitempty"`
```

Comment:
I guess those things are not related to this PR and should be removed, or? (same for the changes in the `v1alpha1` package)

Reply:
Good point. Lingering from the storage work. Removed.